modules/local/samplesheet_check.nf

```diff
@@ -17,7 +17,7 @@ process SAMPLESHEET_CHECK {
     when:
     task.ext.when == null || task.ext.when
 
-    script: // This script is bundled with the pipeline, in nf-core/staramr/bin/
+    script: // This script is bundled with the pipeline, in phac-nml/staramrnf/bin/
```

Comment:
I do not believe this file is used and is left over from the nf-core template. Can you remove it?

Reply:
Forgot to remove the python script with the subworkflow d2e71ca